File: Domain/Models/Company.cs
namespace Domain.Models
{
    public class Company
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Phonenumber { get; set; }
        public string Email { get; set; }
        public string Logo { get; set; }
        public string Address { get; set; }
        public bool IsVisibility { get; set; }

        public string LocationId { get; set; }
        public Location Location { get; set; }

        public List<Worker> Workers { get; set; }
        public List<Service> Services { get; set; }
        public List<CompanyOpenHours> CompanyOpenHours { get; set; }
        public List<CompanyImage> CompanyImages { get; set; }
        public List<CompanyLike> LikedByUsers { get; set; }

        public Company()
        {
            Id = Guid.NewGuid().ToString();
            Workers = new List<Worker>();
            Services = new List<Service>();
            CompanyOpenHours = new List<CompanyOpenHours>();
            CompanyImages = new List<CompanyImage>();
            LikedByUsers = new List<CompanyLike>();
        }

    }
}

File: WebBeautyBook/ClientApp/src/app/models/Company.ts
import { CompanyImage } from "./CompanyImage"
import { CompanyOpenHours } from "./CompanyOpenHours"
import { Location } from "./Location"
import { Service } from "./Service"

export class Company{

    id:string = ''
    name:string = ''
    phonenumber:string = ''
    email: string = ''
    logo: string = ''
    address : string = ''
    isVisibility: boolean = false
    locationId:string = ''
    
    location: Location = new Location
    services: Service[] = []
    companyOpenHours: CompanyOpenHours[] = []
    companyImages: CompanyImage[] = []

    /** isFavorite — local property is always "false" by default.
     * @remarks API does not return this property! */ 
    isFavorite:boolean = false
}